src/shared/edit-util.c

```diff
@@ -97,25 +99,25 @@ int create_edit_temp_file(
                         "%s%s"
                         "\n"
                         "%s\n",
-                        new_path,
+                        target_path,
                         strempty(marker_start),
```

Comment:
If the above comment is correct, then `strempty()` can be dropped, though not a big matter.